libs/atqr/domain/src/lib/deadline-monitor-service.ts

```diff
@@ -4,13 +4,12 @@ import { DeadlineInterface } from './deadline-interface';
 
 
 export class DeadlineMonitorService {
-  private _ongoing: DeadlineInterface[] = [];
   private _cron: CronJob;
 
   constructor() {
     this._cron = new CronJob(
       '* * * 0 * *',
-      this.dispatchEvent,
+      this.callEmail,
```

Comment:
This branch and PR are about the challenge entity test, deadline interface changes should not have been made here. Please stash and then remove these changes.